dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -69,7 +66,8 @@ public void testGenerateDashboard()
         Assert.assertTrue(fileName + " is missing", Files.isRegularFile(componentReport));
         try {
           Document report = builder.build(componentReport.toFile());
-          // TODO test the content of the report
+          Nodes updates = report.query("//li");
+          Assert.assertTrue("didn't find updates", updates.size() > 2);
```

Comment:
It's intended that this test is non-hermetic, right?

Reply:
You mean, it's intended that it talks to Maven central? Yes, that is intended.

Reply:
What I really meant was that this test will break when there's an update published.